src/app/dataset/workflow/workflow.component.ts

```diff
@@ -65,6 +66,7 @@ export class WorkflowComponent implements OnInit {
   workflowForm: FormGroup;
   isSaving = false;
   gapInSequence = false;
+  subUpdate: Subscription;
```

Comment:
Can' t you do this using the `SubscriptionManager`?

Reply:
Done